### DISCUSSION TITLE
Passing down styles via props.

### DISCUSSION THREAD

**Discussion by jim4067** [chakra-ui/panda#2461] [Q&A] [answered]
I have a component that I'd like to receive styles from another component file.  

```ts
export function Dummy{
	label,
	style,
}: TitleProps) {
	return (
		<h1
			className={css(
				{
					fontSize: "xxLarge"
					// ...style,
				},
				style
			)}
		>
			{label}
		</h1>
	);
}
```

For some reason when either using destructing or merging the styles, the classes show up in my markdown but arent being applied

In the HTML

![image](https://github.com/chakra-ui/panda/assets/30603522/0dce36dd-b6cb-4f7c-bda5-12f34147c4fd)

As shown below, the actual component is missing the blue color and text-transform properties I wrote

![image](https://github.com/chakra-ui/panda/assets/30603522/fcf64029-12df-4b32-8b62-75db9209dd3a)

**Comment by astahmer** [comment, marked as answer]
Panda relies on static analysis, [property renaming](https://panda-css.com/docs/guides/dynamic-styling#property-renaming) is one of its limitations
you can use [the special `css` property for that](https://panda-css.com/docs/concepts/merging-styles#:~:text=Tip%3A%20Panda%20extracts%20the%20css%20prop%20from%20the%20JSX%20component) (instead of `style`)
